scripts/update_inventory.py

```diff
@@ -0,0 +1,35 @@
+import docopt
+
+import teuthology
+import teuthology.config
+import teuthology.lock
+import teuthology.orchestra.remote
```

Comment:
Are all these imports needed? Looks like you're using the full path to the modules being used below.